modules/pytket-pyquil/setup.py

```diff
@@ -41,7 +41,6 @@
     install_requires=[
         "pytket ~= 1.3",
         "pyquil ~= 3.1",
-        "qcs-api-client < 0.20.14",
```

Comment:
For me this looks like the CI run has not build the docs?

Reply:
(Try running `pip install -r requirements.txt` in the pytket repo and see the error message.)